guides/source/5_2_release_notes.md

```diff
@@ -68,36 +159,128 @@ Please refer to the [Changelog][railties] for detailed changes.
 *   Deprecate `capify!` method in generators and templates.
     ([Pull Request](https://github.com/rails/rails/pull/29493))
 
-*   Deprecated passing the environment's name as a regular argument to the
-    `rails dbconsole` and `rails console` commands.
-    ([Pull Request](https://github.com/rails/rails/pull/29358))
+*   Passing the environment's name as a regular argument to the
+    `rails dbconsole` and `rails console` commands is deprecated.
+    The `-e` option should be used instead.
+    ([Commit](https://github.com/rails/rails/commit/48b249927375465a7102acc71c2dfb8d49af8309))
 
-*   Deprecated using subclass of `Rails::Application` to start the Rails server.
+*   Deprecate using subclass of `Rails::Application` to start the Rails server.
     ([Pull Request](https://github.com/rails/rails/pull/30127))
 
-*   Deprecated `after_bundle` callback in Rails plugin templates.
+*   Deprecate `after_bundle` callback in Rails plugin templates.
     ([Pull Request](https://github.com/rails/rails/pull/29446))
 
 ### Notable changes
 
-ToDo
+*   Namespace error pages' CSS selectors to stop the styles from bleeding
+    into other pages when using Turbolinks.
+    ([Pull Request](https://github.com/rails/rails/pull/28814))
+
+*   Added a shared section to `config/database.yml` that will be loaded for
+    all environments.
+    ([Pull Request](https://github.com/rails/rails/pull/28896))
+
+*   Allow irb options to be passed from `rails console` command.
+    ([Pull Request](https://github.com/rails/rails/pull/29010))
+
+*   Add `railtie.rb` to the plugin generator.
+    ([Pull Request](https://github.com/rails/rails/pull/29576))
+
+*   Clear screenshot files in `tmp:clear` task.
+    ([Pull Request](https://github.com/rails/rails/pull/29534))
+
+*   Allow mounting the same engine several times in different locations.
+    ([Pull Request](https://github.com/rails/rails/pull/29662))
```

Comment:
Not sure if it's intentional, but this one was backported to Rails 5.1.3.

Reply:
I removed this entry a489cc81b6aa1187b169ddccc92f4c077fe0408f. Thanks!

Reply:
I'll try to catch more backported changes this week since I didn't pay attention to this attribute.
I want to improve a bit this guide by extracting info about CSP into docs(https://github.com/rails/rails/pull/32222#issuecomment-372269242) so will combine these in one patch.
Thank you.